<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Training</class>
 <widget class="QDialog" name="Training">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basketball Team Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="8">
    <widget class="QLabel" name="cost2">
     <property name="text">
      <string>Total E cost</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLCDNumber" name="lcdNumber_2"/>
   </item>
   <item row="12" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Player6</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_2p">
     <property name="text">
      <string>Pos2</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4" rowspan="3" colspan="2">
    <widget class="QGraphicsView" name="graphicsView_9"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelp">
     <property name="text">
      <string>Pos1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="8">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QLabel" name="label_4p">
     <property name="text">
      <string>Pos4</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" rowspan="3" colspan="2">
    <widget class="QGraphicsView" name="graphicsView_5"/>
   </item>
   <item row="11" column="4" colspan="2">
    <widget class="QProgressBar" name="progressBar_7">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="2">
    <widget class="QProgressBar" name="progressBar_6">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="12" column="8">
    <widget class="QLCDNumber" name="lcdNumber_15"/>
   </item>
   <item row="12" column="6">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Player8</string>
     </property>
    </widget>
   </item>
   <item row="6" column="7">
    <widget class="QLCDNumber" name="lcdNumber_4"/>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar_5">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="8" column="8">
    <widget class="QLabel" name="cost1">
     <property name="text">
      <string>Energy cost / p</string>
     </property>
    </widget>
   </item>
   <item row="9" column="8">
    <widget class="QLCDNumber" name="lcdNumber_13"/>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Player5</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" rowspan="3" colspan="2">
    <widget class="QGraphicsView" name="graphicsView_7"/>
   </item>
   <item row="6" column="5">
    <widget class="QLCDNumber" name="lcdNumber_3"/>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="label_3p">
     <property name="text">
      <string>Pos3</string>
     </property>
    </widget>
   </item>
   <item row="12" column="4">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Player7</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_5p">
     <property name="text">
      <string>Pos5</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLCDNumber" name="lcdNumber_5"/>
   </item>
   <item row="0" column="6" rowspan="4" colspan="2">
    <widget class="QGraphicsView" name="graphicsView_4"/>
   </item>
   <item row="4" column="6" colspan="2">
    <widget class="QProgressBar" name="progressBar_4">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QProgressBar" name="progressBar_2">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Player1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Player3</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Player2</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="4" colspan="2">
    <widget class="QGraphicsView" name="graphicsView_2"/>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QProgressBar" name="progressBar_3">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="4" colspan="2">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="0" column="4" rowspan="4" colspan="2">
    <widget class="QGraphicsView" name="graphicsView_3"/>
   </item>
   <item row="4" column="8">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Player4</string>
     </property>
    </widget>
   </item>
   <item row="19" column="5">
    <widget class="QLCDNumber" name="lcdNumber_11"/>
   </item>
   <item row="14" column="8">
    <widget class="QLabel" name="cost3">
     <property name="text">
      <string>Money cost</string>
     </property>
    </widget>
   </item>
   <item row="19" column="4">
    <widget class="QLabel" name="label_11p">
     <property name="text">
      <string>Pos11</string>
     </property>
    </widget>
   </item>
   <item row="19" column="3">
    <widget class="QLCDNumber" name="lcdNumber_10"/>
   </item>
   <item row="18" column="8">
    <widget class="QLCDNumber" name="lcdNumber_16"/>
   </item>
   <item row="14" column="6" rowspan="3" colspan="2">
    <widget class="QGraphicsView" name="graphicsView_12"/>
   </item>
   <item row="19" column="2">
    <widget class="QLabel" name="label_10p">
     <property name="text">
      <string>Pos10</string>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="label_9p">
     <property name="text">
      <string>Pos9</string>
     </property>
    </widget>
   </item>
   <item row="18" column="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Player10</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar_9">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="14" column="2" rowspan="3" colspan="2">
    <widget class="QGraphicsView" name="graphicsView_8"/>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Player9</string>
     </property>
    </widget>
   </item>
   <item row="17" column="4" colspan="2">
    <widget class="QProgressBar" name="progressBar_11">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="13" column="7">
    <widget class="QLCDNumber" name="lcdNumber_8"/>
   </item>
   <item row="14" column="4" rowspan="3" colspan="2">
    <widget class="QGraphicsView" name="graphicsView_11"/>
   </item>
   <item row="13" column="6">
    <widget class="QLabel" name="label_8p">
     <property name="text">
      <string>Pos8</string>
     </property>
    </widget>
   </item>
   <item row="17" column="2" colspan="2">
    <widget class="QProgressBar" name="progressBar_10">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="18" column="4">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Player11</string>
     </property>
    </widget>
   </item>
   <item row="19" column="6">
    <widget class="QLabel" name="label_12p">
     <property name="text">
      <string>Pos12</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QLabel" name="label_6p">
     <property name="text">
      <string>Pos6</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" rowspan="3" colspan="2">
    <widget class="QGraphicsView" name="graphicsView_6"/>
   </item>
   <item row="17" column="6" colspan="2">
    <widget class="QProgressBar" name="progressBar_12">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="13" column="4">
    <widget class="QLabel" name="label_7p">
     <property name="text">
      <string>Pos7</string>
     </property>
    </widget>
   </item>
   <item row="17" column="8">
    <widget class="QLabel" name="cost4">
     <property name="text">
      <string>Bugdet</string>
     </property>
    </widget>
   </item>
   <item row="19" column="7">
    <widget class="QLCDNumber" name="lcdNumber_12"/>
   </item>
   <item row="15" column="8">
    <widget class="QLCDNumber" name="lcdNumber_14"/>
   </item>
   <item row="19" column="1">
    <widget class="QLCDNumber" name="lcdNumber_9"/>
   </item>
   <item row="13" column="5">
    <widget class="QLCDNumber" name="lcdNumber_7"/>
   </item>
   <item row="13" column="3">
    <widget class="QLCDNumber" name="lcdNumber_6"/>
   </item>
   <item row="20" column="8">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="18" column="6">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Player12</string>
     </property>
    </widget>
   </item>
   <item row="11" column="6" colspan="2">
    <widget class="QProgressBar" name="progressBar_8">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="8" column="6" rowspan="3" colspan="2">
    <widget class="QGraphicsView" name="graphicsView_10"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
